### ISSUE TITLE
quotes.js doesn't choose from all chunks

### ISSUE THREAD

**Issue by maxcai314** [r2dev2/cryptoduel#4]
in line 13-17 of src/js/quotes.js, a random quote chunk is chosen by taking the floor of multiplying`Math.random()` to `amountChunks - 1`.
```
  const chunk = fetch(
    `./quotes/${Math.floor(Math.random() * (amountChunks - 1))}.json`
  )
    .then((r) => r.json())
    .then(shuffleArray);
```
The `Math.random()` function returns a floating-point number in the bounds of `[0, 1)`. This means that the method will return a value from 0 to 1, but will never returns a value of 1. By multiplying this by `amountChunks - 1`, floor of the result will never be equal to `amountChunks - 1`, because `Math.random()` is always less than 1. This means that the last chunk of quotes will never be selected.

To fix this, we should multiply `Math.random()` by `amountChunks` instead, so that the floor of the product will be (almost) perfectly distributed between integer values ranging from `0` to `amountChunks - 1`.

Therefore, I suggest:
```
  const chunk = fetch(
    `./quotes/${Math.floor(Math.random() * (amountChunks))}.json`
  )
    .then((r) => r.json())
    .then(shuffleArray);
```

**Comment by maxcai314** [comment]
#5

**Comment by maxcai314** [comment]
#5

**Comment by r2dev2** [comment]
I originally excluded the last chunk as it has less quotes than the rest of the chunks. However, looking more closely, I see that last chunk has `369` quotes while the other chunks have `500` so it is feasible to use the last chunk.

**Comment by r2dev2** [comment]
Fixed with merge of #5 

Thank you for contributing!

Also, just an fyi, in your git commit for this fix, you used an email which you don't have registered in your github account. As a result, it may not show properly that you contributed to this repository. To fix this, you can either add your email to your github account or change your gitconfig to use your github email.
